python/composio/tools/base/abs.py

```diff
@@ -426,6 +426,18 @@ def setup_children(obj: t.Type["Tool"]) -> None:
             if hasattr(obj, "logo"):
                 setattr(trigger, "logo", getattr(obj, "logo"))
 
+    @staticmethod
+    def _get_description(obj) -> str:
```

Comment:
The `_get_description` method should include type hints for its parameters and a docstring explaining its functionality:

```python
@staticmethod
def _get_description(obj: t.Type["Tool"]) -> str:
    """
    Extracts and formats the description from a tool object's docstring or display name.
    
    Args:
        obj: The tool object to extract description from
        
    Returns:
        str: Formatted description with cleaned whitespace and joined multiple lines
    """
```